src/fft_small/mpn_mul.c

```diff
@@ -1388,31 +1385,29 @@ typedef struct mod_fft_worker_struct {
 void mod_fft_worker_func(void* varg)
```

Comment:
Now that you have made changes to this function: Is there any way to test this function? We have been unable to reach this function in the test code.

Reply:
The only way to test this currently is to get an operand size bigger than any bn_bound in the list of profiles. This isn't going to happen.  If you had a configure option for testing, you could do something random like
```
    if ((FLINT_TESTING && (bn%7) < 3) ||
        FLINT_UNLIKELY(bn > R->profiles[i].bn_bound))
    {
        P->np = n_max(n_min(P->nthreads, 8), 4);
        P->bits = crt_data_find_bits(R->crts + P->np - 1, bn);
        P->to_ffts = NULL;
        return;
    }
```

Reply:
If/when I add the `mpn_ctx_mpn_mul_mid_approx` and `mpn_ctx_mpn_mod_bpnm1`, it might use the slow branch.